config/routes.rb

```diff
@@ -7,9 +7,9 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-  root "posts#index"
+  root "users#show_followed_posts"
 
-  resources :users, only: %i[edit update index]
+  resources :users, only: %i[edit update index show_followed_posts]
```

Comment:
There's no need of defining a new path + method for this. Let's just redefine the index.